jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/skaffold/SkaffoldSyncParameters.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.gradle.skaffold;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.util.Collections;
+import java.util.Set;
+import java.util.stream.Collectors;
+import javax.inject.Inject;
+import org.gradle.api.Project;
+import org.gradle.api.tasks.Internal;
+
+public class SkaffoldSyncParameters {
+  private final Project project;
+
+  private Set<Path> excludes = Collections.emptySet();
+
+  @Inject
+  public SkaffoldSyncParameters(Project project) {
+    this.project = project;
+  }
+
+  /**
+   * Get the excludes directive for sync functionality in skaffold.
+   *
+   * @return a set of absolute paths
+   */
+  @Internal
+  public Set<Path> getExcludes() {
+    // Gradle warns about @Input annotations on File objects, so we have to expose a getter for a
+    // String to make them go away.
```

Comment:
Does this comment apply when we use Path?

Reply:
Oh yeah this comment was about inputs, copy pasta :disappointed:  Needs to be removed.